Mesh_3/include/CGAL/Mesh_3/Slivers_exuder.h

```diff
@@ -195,7 +195,10 @@ class Slivers_exuder_base
   Tet_priority_queue cells_queue_;
 };
 
-#ifdef CGAL_LINKED_WITH_TBB
+
+  //  The debug version of the VC++ testsuite gets a timeout when using TBB, so let's disable it
+#if defined( CGAL_LINKED_WITH_TBB ) && ( (! defined (BOOST_MSVC)) || ( defined (BOOST_MSVC)  && defined( _DEBUG ) && (! defined ( CGAL_TEST_SUITE ) ) ) )
```

Comment:
The condition looks wrong. You want to deactivate it when all those macros are defined:
  - `BOOST_MSVC`
  - `_DEBUG`
  - `CGAL_TEST_SUITE`

That means that you want to enable it as soon as one of those three macros is undefined:
```c++
#if defined( CGAL_LINKED_WITH_TBB ) && ( !defined(BOOST_MSVC) || !defined(_DEBUG) || !defined(CGAL_TEST_SUITE) )
```